active/RadioButtonsAPI.md

```diff
@@ -0,0 +1,105 @@
+# Background
+
+RadioButtons is our answer to enabling better keyboarding and narrator support for RadioButtons that need to be presented in a related list.
+
+Currently, the only way to group RadioButtons is through using a StackPanel and individually labeling each RadioButton through its GroupName property in order for narrator to read out the options somewhat understandably.
+
+However, using that method still left gaps in the narrator experience and StackPanel does not behave the way a user would expect when navigating the list via keyboard - RadioButtons was introduced to help solve those issues.
+
+# Description
+
+RadioButtons is a class extension of ListView that enables you to create related lists of RadioButtons easily, while also fully supporting keyboarding and narrator functionality.
+
+# Examples
+
+The following example demonstrates how to create a small collective list of RadioButtons using RadioButtons.
+
+```xaml
+<RadioButtons Header="App Mode" SelectedIndex="2">
+    <x:String>Dark</x:String>
+    <x:String>Light</x:String>
+    <x:String>Windows Default</x:String>         
+</RadioButtons>
+```
+
+# API Details
+
+### RadioButtons
+
+```
+namespace Microsoft.UI.Xaml.Controls
+{
+[webhosthidden]
+[contentproperty("Items")]
+unsealed runtimeclass RadioButtons : Windows.UI.Xaml.Controls.Control
+{
+    RadioButtons();
+
+    Object ItemsSource;
+    Windows.Foundation.Collections.IVector<Object> Items{ get; };
+    Windows.UI.Xaml.DataTemplate ItemTemplate;
+
+    Windows.UI.Xaml.DependencyObject ContainerFromItem(Object item);
+    Windows.UI.Xaml.DependencyObject ContainerFromIndex(Int32 index);
+
+    Int32 SelectedIndex;
+    Object SelectedItem;
+    event Windows.UI.Xaml.Controls.SelectionChangedEventHandler SelectionChanged;
+
+    Int32 MaximumColumns;
+    Object Header;
+
+    static Windows.UI.Xaml.DependencyProperty ItemsSourceProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ItemsProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ItemTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedIndexProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedItemProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MaximumColumnsProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeaderProperty{ get; };
+}
+}
+```
+
+### RadioButtons Automation Pier
+
+```
+namespace Microsoft.UI.Xaml.Automation.Peers
+{
+[webhosthidden]
+unsealed runtimeclass RadioButtonsListViewItemAutomationPeer : Windows.UI.Xaml.Automation.Peers.ListViewItemAutomationPeer
+{
+    RadioButtonsListViewItemAutomationPeer(RadioButtonsListViewItem owner);
+}
+}
+```
+
+### RadioButtons Primitives
+
+```
+namespace Microsoft.UI.Xaml.Controls.Primitives
+{
+[webhosthidden]
+unsealed runtimeclass RadioButtonsListView : Windows.UI.Xaml.Controls.ListView
```

Comment:
Is *.Primitives the right place for this API? It contains things like ButtonBase and Popup.

Reply:
It's an implementation detail of the control itself but needs to be public in case you retemplate. Primitives is where we've put things like this in the past but I'm fine if you think another namespace is appropriate?

Reply:
I don't have strong pushback (or a better idea 😅) it just seemed like a more complex control than some of the other types found in that namespace.

Reply:
+1 to Primitives. (This Primitives pattern has issues, but nice to keep them all in one place and potentially fix them at the same time.)